dcos.yml

```diff
@@ -1,6 +1,7 @@
 ---
 - name: Wait for instances to become reachable
   hosts: all
+  gather_facts: no
   tasks:
     - name: Wait for instances to become reachable
       wait_for_connection:
```

Comment:
the line below this which contains `sleep: 10` needs to be updated to `delay: 10`

Reply:
and additional `sleep` should be kept as well I would say.